### PR TITLE
fix: #142 rp adapter: send consent VC to issuer adapter

### LINKED ISSUE #142: rp adapter: send consent VC to the Issuer Adapter
also no need to send the presentation-definitions.

Just need to send the ~~user's peer DID and the~~ user's consent VC

### PR DESCRIPTION
**Pull request by llorllale**
fixes #142 

depends on #144 

Request to the issuer adapter:

```json
{
  "@id": "12345",
  "@type": "https://didcomm.org/present-proof/2.0/request-presentation",
  "formats": [{
    "attach_id": "123",
    "format": "trustbloc/UserConsentVerifiableCredential@0.1.0",
  }],
  "request_presentations~attach": [{
    "@id": "123",
    "mime-type": "application/ld+json",
    "data": {
        "base64": "base64Encode(vp with user consent VC)"
    }
  }]
}
```

Signed-off-by: George Aristy <george.aristy@securekey.com>

### CODE REVIEW

**Comment by llorllale** [comment]
I don't know how to increase test coverage on this one:

* missed coverage on lines operations.go#L847-862 depend on errors while converting `consentVC` to a marshalled verifiable presentation
* this corresponds to the framework's `Credential.Presentation()` and `Presentation.MarshalJSON()` respectively
* however, `consentVC` at this point is already valid as per the output of `parseWalletResponse()` on operations.go#L779

It's basically unreachable code at this point.

File: pkg/restapi/rp/operation/credentials_test.go [hunk 1]
```diff
@@ -106,7 +106,7 @@ func TestParseIssuerResponse(t *testing.T) {
 		result, err := parseIssuerResponse(nil, &presentproof.Presentation{
 			Formats: []presentproofsvc.Format{{
 				AttachID: attachID,
-				Format:   presentationSubmissionFormat,
+				Format:   "",
```
Comment (soluchok) [inline pkg/restapi/rp/operation/credentials_test.go:109]: remove empty

Comment (llorllale) [reply pkg/restapi/rp/operation/credentials_test.go]: @soluchok fixed

File: pkg/restapi/rp/operation/operations_test.go [hunk 2]
```diff
@@ -1914,7 +1915,7 @@ func TestHandleIssuerPresentationMsg(t *testing.T) {
 		msg := service.NewDIDCommMsgMap(&presentproof.Presentation{
 			Formats: []presentproofsvc.Format{{
 				AttachID: uuid.New().String(),
-				Format:   presentationSubmissionFormat,
+				Format:   "",
```
Comment (soluchok) [inline pkg/restapi/rp/operation/operations_test.go:1918]: nit: same and there are two more

Comment (llorllale) [reply pkg/restapi/rp/operation/operations_test.go]: removed, thanks

**Pull request merged**